target/linux/brcm47xx/image/Makefile

```diff
@@ -403,7 +419,10 @@ endef
 TARGET_DEVICES += linksys-wrt54g
 
 define Device/linksys-wrt54gs
-  DEVICE_TITLE := Linksys WRT54GS
+  DEVICE_MODEL := WRT54GS
+  DEVICE_VARIANT := v1/v2/v3
+  DEVICE_ALT0_MODEL := WRT54G-TM
```

Comment:
Ah, crap. You need a DEVICE_ALT0_VENDOR here.
After that, it is ready for merge :-)